pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1320,7 +1320,8 @@ class HdRprApiImpl {
     }
 
     void UpdateCamera() {
-        if ((m_dirtyFlags & ChangeTracker::DirtyCamera) == 0) {
+        if ((m_dirtyFlags & ChangeTracker::DirtyCamera) == 0 &&
```

Comment:
Should be this an || ?

Reply:
No, if both flags are not set we return from function

Reply:
Sorry, yes.  reverse logic.